xcode/Subconscious/SubconsciousTests/Tests_UserDefaultProperty.swift

```diff
@@ -46,6 +46,6 @@ final class Tests_UserDefaultProperty: XCTestCase {
         test.isToggled = false
         test.isToggled = false
         test.isToggled = false
-        XCTAssertEqual(count, 2)
+        XCTAssertEqual(count, 1)
```

Comment:
Turns out the reason we were getting 2 counts is because the value wasn't being read from the backing store on construction. Test updated. 😨